node/src/service.rs

```diff
@@ -413,32 +391,27 @@ fn start_consensus(
         client.clone(),
     );
 
-    let params = AuraParams {
+    let params = BasicAuraParams {
```

Comment:
Why is it going back to the basic aura config rather than the lookahead for async backing?

Reply:
This previously didn't compile. I changed it to to what the template had and it compiled and async backing was working. In addition, the next upgrade to [1.11 changes it back to `AuraParams`](https://github.com/paritytech/polkadot-sdk/blob/12ab31508cf8f03e6cc233722e535c383dc97c41/templates/parachain/node/src/service.rs#L197).

To my opinion, we should have this template one to one to the template in the polkadot sdk. This to simplify the maintenance process and being sure of correctness. If we make slight changes we only shoot ourselves in the foot because we need to do more work.

Reply:
By that logic then we need to revert all the recent fixes that were made to this repo, just to match the cumulus template.

It would be better the upstream these improvements than revert. These templates are our responsibility now and we should be putting out quality and feeding it back where possible.

Removing async backing just to readd it in next version seems silly to me.